### PR TITLE
Swagger2 version of 0.4.0 api

### PR DESCRIPTION
**Pull request by hgs-msmith**
Here is a swagger 2 version of the api that generates working code with the Swagger Tools. I believe there may be one or two bugs in the OpenAPI 3 version that this backport has exposed. I am working on those next.

### CODE REVIEW

File: api-spec/WFS3core+STAC-swagger2.yaml [hunk 1]
```diff
@@ -0,0 +1,779 @@
...
+          - Point
+          - MultiPoint
+          - LineString
+          - MultiLineString
+          - Polygon
+          - MultiPolygon
+          - GeometryCollection
+  searchBody:
+    description: The search criteria
+    allOf:
```
Comment (hgs-trutherford) [inline api-spec/WFS3core+STAC-swagger2.yaml]: Does Swagger 2 support allOf?

**Comment by hgs-msmith** [comment]
Yes, but only for object composition.

Michael Smith
Chief Architect, Commercial Geospatial Products
space and intelligence systems / HARRIS CORPORATION
Office: +1-303-402-4667
harris.com<http://www.harris.com/>  / Michael.Smith1@harris.com<mailto:Michael.Smith1@harris.com>
385 Interlocken Crescent, Suite 300 / Broomfield, Colorado 80021 / USA
[Harris_wR_2color_72dpi.jpg]<http://www.harris.com/>

**Comment by hgs-msmith** [comment]
Closed dupe PR (#99) - not sure how that happened.

**Comment by hgs-msmith** [comment]
I can add a standalone as well. I'll merge it all the way down to master.

**Pull request merged**